test/torch/tensors/test_poly.py

```diff
@@ -0,0 +1,60 @@
+import syft 
+from syft.frameworks.torch.tensors.Polynomial import PolynomialTensor
+
+import math
```

Comment:
we probably should be comparing to the torch.nn.functional definitions of these functions rather than the math ones as we are trying to preserve feature parity with torch